package-testing/php-sdk-relay/src/index.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Eppo\SDKTest\AssignmentHandler;
+use Eppo\SDKTest\BanditHandler;
+use Eppo\SDKTest\Config;
+use Eppo\SDKTest\TestLogger;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+use Eppo\EppoClient;
+use Slim\Http\Response;
+
+$dotenv = Dotenv\Dotenv::createImmutable(__DIR__ . '/../');
+$dotenv->safeLoad();
+
+$config = new Config();
+
+$testLogger = new TestLogger();
+
+$eppoClient = EppoClient::init(
+    $config->apiKey,
+    $config->apiServer,
+    $testLogger
+);
+
+$app = AppFactory::create();
+
+$app->get('/', function (Request $request, Response $response, array $args) {
```

Comment:
Nice thinking! Could be useful for docker healthcheck too

Reply:
✔️